body.tex

```diff
@@ -1,21 +1,341 @@
+During the measurement stage, for each tract and band, the matched catalog computed and stored in the preparation stage is loaded into memory and used to compute the RMS scatter of fluxes for a single astrophysical object. 
+In the final summary stage, the measurements for the ensemble of individual tracts computed in the measurement stage are loaded to compute a median summary statistic per band. 
+This final summary statistic characterizes the overall performance for the dataset per band and is stored as an \texttt{lsst.verify} object in the Butler. 
+
+\subsection{Processing Pipelines} \label{ssec:pipelines}
+
+Pipelines, configurations in the form of YAML files, are used to define the \faro metrics to be computed together with the  detailed execution parameters for metric calculations. 
+The \faro package provides a set of pre-defined pipelines, defined per analysis context, for many common configurations of computing metrics. 
+\faro pipelines can be run as an after burner to the LSST Science Pipelines or can be interspersed with science pipelines processing to compute , as long as the necessary input data products have been computed and are stored in the Butler.  
+For example, all single-visit metrics can already be computed once the single-frame processing steps have completed. 
+This enables us to promptly assess performance and take any corrective action early. 
+Figure \ref{fig:faro_pipeline} shows an example of a \faro pipeline describing the measurement stage of the computation of three performance metrics  using a per-tract matched catalog analysis context.
+Pipelines can be built hierarchically, with a high-level pipeline calling lower-level pipelines, as shown in figure \ref{fig:faro_pipeline_hierarchy}.
+
+\begin{figure}[!ht]
+  \lstset{language=YAML}
+  \begin{lstlisting}
+description: Compute metrics from matched catalogs
+tasks:
+  PA1:
+    class: lsst.faro.measurement.TractMatchedMeasurementTask
+    config:
+      connections.package: validate_drp
+      connections.metric: PA1
+      python: |
+        from lsst.faro.measurement import PA1Task
+        config.measure.retarget(PA1Task)
+  PF1_design:
+    class: lsst.faro.measurement.TractMatchedMeasurementTask
+    config:
+      connections.package: validate_drp
+      connections.metric: PF1_design_gri
+      python: |
+        from lsst.faro.measurement import PF1Task
+        config.measure.retarget(PF1Task)
+        config.measure.threshPA2 = 15.0
+    \end{lstlisting}
+  \caption{A \faro pipeline describing the measurement stage of the computation of three performance metrics per-tract from matched catalogs.}
+  \label{fig:faro_pipeline}
+\end{figure}
+
+\begin{figure}[!ht]
```

Comment:
I think the description in the text is sufficient for describing the organization, and this snippet of yaml doesn't really add much.

Reply:
removed